PSKoans/Koans/Constructs and Patterns/AboutErrorHandling.Koans.ps1

```diff
@@ -18,60 +18,76 @@ param()
 #>
 
 Describe 'ErrorRecord' {
-    BeforeAll {
-        $ErrorRecord = try {
-            throw "A challenge to the sky!"
+    Context '$Error' {
+        BeforeAll{
+            <#
+                $Error is an automatic variable that contains a list of recent errors.
+                By calling $Error.Clear() we ensure past errors to not carry into our testing.
+            #>
+            $Error.Clear()
         }
-        catch {
-            $_
+        It 'sometimes has a reference to the object that caused the error' {
+            # But not always!
+            $Error[0].TargetObject | Should -BeNullOrEmpty
+    
+            # Non-terminating error behaviour can be adjusted with the -ErrorAction parameter.
+            Get-Item -Path "TestDrive:\This_Shouldn't_Exist" -ErrorAction SilentlyContinue
+    
+            <#
+                $Error is an automatic variable that contains a list of recent errors. $Error[0] is always the most
+                recent error. Even if an error is silenced with -ErrorAction SilentlyContinue, it is still recorded
+                in $Error.
```

Comment:
The first sentence here is a duplicate of the above mention where $Error.Clear() is referenced.

Reply:
Updated.